seclang/parser.go

```diff
@@ -76,19 +76,29 @@ func (p *Parser) FromString(data string) error {
 	scanner := bufio.NewScanner(strings.NewReader(data))
 	var linebuffer = ""
 	pattern := regexp.MustCompile(`\\(\s+)?$`)
+	inQuotes := false
 	for scanner.Scan() {
 		p.currentLine++
-		line := scanner.Text()
-		linebuffer += strings.TrimSpace(line)
+		line := strings.TrimSpace(scanner.Text())
```

Comment:
This change requires a unit test.

Reply:
CRS tests are passing, but we should definitely add more tests for this.